app/controllers/APIController.scala

```diff
@@ -6,11 +6,13 @@ import io.circe.syntax.EncoderOps
 import javax.inject._
 import play.api._
 import play.api.mvc._
+import DAO.BakeryDB
 
 @Singleton
 class APIController @Inject() (
     val environment: Environment,
-    val controllerComponents: ControllerComponents
+    val controllerComponents: ControllerComponents,
```

Comment:
do you need to have `environment` and `controllerComponents` declared as `val`? can they just be parameters?